### DISCUSSION TITLE
[CLI] Is there a feature to build without the output(.next)?

### DISCUSSION THREAD

**Discussion by zudydy** [vercel/next.js#68178] [Help]
### Summary

While I'm working locally by running the `next dev` command, there are times when I want to check through the build to see if all the current codes are working properly.

However, if you run the `next build` while running the `next dev`, the `.next` directory will be overwritten, causing conflicts on the local output screen. Each time I have to erase the `.next` directory and run the `next dev` again.

So is there a feature or option to check if the build ran properly without generating the build output?

Thx :)

**Comment by Juneezee** [comment]
I don't think it is possible to disable the output directory. Next.js needs it to store cache etc.

However, there is a `distDir` option in `next.config.js` that you can use to set a custom build directory, see https://nextjs.org/docs/app/api-reference/next-config-js/distDir.
